plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -36,11 +36,18 @@
             GET /1.0/containers/<name>
             U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersname)
             are different, they this module tries to apply the configurations.
-          - The key starts with 'volatile.' are ignored for this comparison.
           - Not all config values are supported to apply the existing container.
             Maybe you need to delete and recreate a container.
         type: dict
         required: false
+    ignore_volatile_options:
+            description:
```

Comment:
```suggestion
        description:
```
Same for the other lines.

Reply:
done